packages/core-js/test/utils/misc/dates.test.js

```diff
@@ -0,0 +1,35 @@
+const { ok, equal, throws } = require('assert/strict');
+const { formatDate, getUnixTimestamp, daysToSeconds } = require('../../../utils/misc/dates');
+
```

Comment:
Added a few date related utilities to `get-dates.js`, and renamed it to `dates.js`.